flipperdevices/Flipper-Android-App
# Remote controls-better ui

**Background**

This PR is designed to update update remote-control buttons so it'll be up to date with IRDB-backend

**Changes**

- Add new ButtonData
- Add new icons
- Add new screen with infrared items by long-pressing on brand

**Test plan**

- Open remote controls
- Long press on brands to open all of it's infrareds
- Select infrareds and see new ui
